### PR TITLE
Update AuthenticationPage.js

### LINKED ISSUE #2133: Custom logo positioning issues
**Describe the bug**
The login button has a negative top-margin that can (depending on the logo size) obscure the custom logo. Additionally, (depending on the logo size) the logo may be left-aligned, when the expectation is for it to be centered.

**To Reproduce**
If you upload a logo that is 300 x 200, the bottom ~40px of the logo will be covered up by the login button. If you upload a logo that is smaller than 300px wide, the logo will align to the left of that 300px container.

**Expected behavior**
The expectation is for the custom logo to be centered and placed above the login button.

**Screenshots**
screen shot 2019-03-01 at 1 16 52 pm

screen shot 2019-03-01 at 1 14 36 pm

**Applicable Versions:**
- netlify-cms 2.5.1
 - Git provider: GitHub
 - OS: macOS 10.14.3
 - Chrome Version 72.0.3626.119 (Official Build) (64-bit)
 - Node.JS version: 11.10.0

**CMS configuration**
No can do, sorry.

**Additional context**
https://github.com/netlify/netlify-cms/blob/636a141a4ab3c4c9a490aea6956b32fec2324e08/packages/netlify-cms-ui-default/src/AuthenticationPage.js

### PR DESCRIPTION
**Pull request by adamcnaglich**
Removed the login button's negative margin to prevent obscuring the logo. Adjusted the custom logos height and width to scale according to the file up to a predefined max-width and max-height so that it can align to the center.

fixes #2133 

**Summary**
Removed the login button's negative margin to prevent obscuring the logo. Adjusted the custom logos height and width to scale according to the file up to a predefined max-width and max-height so that it can align to the center.

**Test plan**
screen shot 2019-03-01 at 1 42 39 pm

screen shot 2019-03-01 at 1 42 20 pm

### CODE REVIEW

**Comment by adamcnaglich** [comment]
No idea what I am doing. Please look at this carefully 😛

File: packages/netlify-cms-ui-default/src/AuthenticationPage.js [hunk 1]
```diff
@@ -13,8 +13,10 @@ const StyledAuthenticationPage = styled.section`
 `;
 
 const CustomIconWrapper = styled.span`
-  width: 300px;
-  height: 200px;
+  width: auto;
+  height: auto;
+  max-width: 300px;
+  mac-height: 200px;
```
Comment (barthc) [inline packages/netlify-cms-ui-default/src/AuthenticationPage.js]: @adamcnaglich do you mean `max-height` ?

Comment (adamcnaglich) [reply packages/netlify-cms-ui-default/src/AuthenticationPage.js]: Dear God

**Comment by adamcnaglich** [comment]
Just checking in on this - does everything look gucci now? The incorrectly aligned logo on the login page has been making a poor first impression to Netlify CMS.

**Comment by barthc** [comment]
@tomrutgers can you test and approve this PR please, it's a first time contributor.

**Comment by tomrutgers** [comment]
I might be missing something but when I'm testing it, the changes do not seem to fix the problem when using larger images:

screenshot

**Comment by adamcnaglich** [comment]
Can you see why the max-height is not being applied to the HTML.COM image?

**Comment by tomrutgers** [comment]
@adamcnaglich It's not applied to the image, it's applied to it's wrapper.

**Pull request closed without merging**